src/Init/Tactics.lean

```diff
@@ -39,8 +39,74 @@ be a `let` or function type.
 syntax (name := intro) "intro" notFollowedBy("|") (ppSpace colGt term:max)* : tactic
 
 /--
-`intros x...` behaves like `intro x...`, but then keeps introducing (anonymous)
-hypotheses until goal is not of a function type.
+Introduces zero or more hypotheses, optionally naming them.
+
+- `intros` is equivalent to repeatedly applying `intro`
+  until the goal is not an obvious candidate for `intro`.
```

Comment:
```suggestion
  until the goal is not a a function type (this includes implications, since these are regarded as functions).
```

Reply:
Do you want this redundancy with the "That is" line that follows?

Reply:
Well, I somehow find it clearer in this version that with the "That is" line that follows: I tend to think that someone who is lost in understanding the expression "function type" (or in seeing why an implication is a term of a function type) would not really be helped by reading `let` or `pi type`. Perhaps we can ask some third opinion?

Reply:
Ok, I've merged the sentences together. Was the "e.g." not enough to explain what a pi type was?

Reply:
I like it this way! Thanks.